README.md

```diff
@@ -175,6 +175,16 @@ Use the `open-metrics` feature flag on the node / client to start an [OpenMetric
 
 The metrics can then be collected using a collector (for e.g. Prometheus) and the data can then be imported into any visualization tool (for e.g., Grafana) to be further analyzed. Refer to this [Guide](./metrics/README.md) to easily setup a dockerized Grafana dashboard to visualize the metrics.
 
-### Archive
 
-The elder-membership agreed, section tree backed implementation of the safe network can be found [here](https://github.com/maidsafe/safe_network_archive)
+
+## Contributing
+
+Please feel free to clone and modify this project. Pull requests are welcome.
+
+## Conventional Commits
+
+We follow the [Conventional Commits](https://www.conventionalcommits.org/) specification for all commits. Make sure your commit messages adhere to this standard.
+
+## License
+
+This project is licensed under the [MIT License](LICENSE).
```

Comment:
I'm not sure we have licensing really set up correctly for this repository and all the crates, but weren't we going to use GPL 3.0 for all `sn` prefixed crates?